src/main/resources/extensions/settings.yaml

```
@@ -31,3 +31,38 @@ spec:
           key: withReplySize
           validation: required
           value: 5
+    - group: avatar
+      label: 头像设置
+      formSchema:
+        - $formkit: checkbox
+          label: 是否启用第三方头像
+          name: enable
+          id: enable
+          key: enable
+          value: false
+        - $formkit: select
+          label: 头像服务提供者
+          if: "$get(enable).value === true"
+          name: provider
+          options:
+            - label: 'Gravatar'
+              value: 'Gravatar'
```

```suggestion
              value: 'gravatar'
```

建议所有 **value** 都采用驼峰命名。